cli/src/main/java/com/devonfw/tools/ide/url/UpdateInitiator.java

```diff
@@ -52,7 +53,12 @@ public static void main(String[] args) {
       System.exit(1);
     }
 
+    UrlFinalReport urlFinalReport = new UrlFinalReport();
+
     UpdateManager updateManager = new UpdateManager(repoPath, expirationTime);
-    updateManager.updateAll();
+    updateManager.updateAll(urlFinalReport);
```

Comment:
Is it good practice to add a new parameter to the `updateAll()` method or should we pass the `urlFinalReport` instance in the constructor of `UpdateManager()` ?

Reply:
I put the urlFinalReport into the UpdateManager Constructor